src/otm_website_tests/core.clj

```diff
@@ -0,0 +1,12 @@
+(ns otm-website-tests.core)
+
+(def base-url (or (System/getenv "OTM_BASE_URL")
+                  "http://localhost:8080"))
+(def http-client-options
+  "Default options for clj-http."
+  {:follow-redirects false
+   :throw-exceptions false})
+
```

Comment:
Adding a ``get`` helper to core removes some bolierplate noise from the facts and makes them more readable

```clojure
(def get [relative-url]
  (client/get (str base-url relative-url) http-client-options))
```

```
(fact "it returns a not found from S3 if trailing slash exists"
    (let [resp (get "/css/")]
      (resp :status) => 404
      (s3-response? resp) => true))
```